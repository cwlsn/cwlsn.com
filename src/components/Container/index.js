import React from 'react'
import styled from 'styled-components'

const Wrapper = styled.div`
	margin: 0 auto;
	padidng: 0;
	max-width: 800px;
`

const Container = ({ children }) => (
	<Wrapper>
		{children}
	</Wrapper>
)

export default Container
