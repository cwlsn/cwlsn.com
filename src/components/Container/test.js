import React from 'react'
import renderer from 'react-test-renderer'
import Container from '.'

test('Container snapshot', () => {
	const tree = renderer
		.create(<Container />)
		.toJSON()

	expect(tree)
		.toMatchSnapshot()
})

test('Container renders children', () => {
	const root = renderer
		.create(<Container><h1>cats</h1></Container>)
		.root

	expect(root.findAllByType('h1').length)
		.toBe(1)

	expect(root.findByProps({ children: 'cats' }).type)
		.toBe('h1')
})
