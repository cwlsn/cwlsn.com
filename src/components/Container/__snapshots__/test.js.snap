// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container snapshot 1`] = `<test-file-stub />`;
